### PR TITLE
Lets admins chose infil faction on give Posted from my Galaxy A32.

### PR DESCRIPTION
**Pull request by Pepsilawn**
None of this is tested i made the entirety of this pr on my phone it took me 20 minutes to figure out how to base my branch on master
Screenshot_20220804-130116_Chrome

### CODE REVIEW

**Comment by Pepsilawn** [comment]
They should add Tab to phones idk how to do them without copypasting them in

**Comment by Pepsilawn** [comment]
something is very wrong

File: fulp_modules/features/antagonists/infiltrators/infiltrator.dm [hunk 1]
```diff
@@ -31,6 +31,17 @@
...
+/datum/antagonist/traitor/infiltrator/admin_add(datum/mind/new_owner, mob/admin)
+	var/list/possible_employers = list( // Should probably be moved somewhere better to make full use of it
+		INFILTRATOR_FACTION_CORPORATE_CLIMBER,
+		INFILTRATOR_FACTION_ANIMAL_RIGHTS_CONSORTIUM,
+		INFILTRATOR_FACTION_GORLEX_MARAUDERS,
+		)
+	employer = input("What affiliation would you like [new_owner] to have?","Affiliation", pick(possible_employers)) in possible_employers
+	message_admins("[key_name_admin(usr)] made [key_name_admin(new_owner)] into \a [employer] [name]")
+	log_admin("[key_name_admin(usr)] made [key_name_admin(new_owner)] into \a [employer] [name]")
+	new_owner.add_antag_datum(src)
```
Comment (JohnFulpWillard) [inline fulp_modules/features/antagonists/infiltrators/infiltrator.dm:43]:
```suggestion
/datum/antagonist/traitor/infiltrator/admin_add(datum/mind/new_owner, mob/admin)
	// Should probably be moved somewhere better to make full use of it
	var/list/possible_employers = list(
		INFILTRATOR_FACTION_CORPORATE_CLIMBER,
		INFILTRATOR_FACTION_ANIMAL_RIGHTS_CONSORTIUM,
		INFILTRATOR_FACTION_GORLEX_MARAUDERS,
	)
	var/choice = input("What affiliation would you like [new_owner] to have?", "Affiliation") as pick(possible_employers) in possible_employers
	if(!choice)
		return
	employer = possible_employers[choice]
	message_admins("[key_name_admin(usr)] made [key_name_admin(new_owner)] into \a [employer] [name]")
	log_admin("[key_name_admin(usr)] made [key_name_admin(new_owner)] into \a [employer] [name]")
	new_owner.add_antag_datum(src)
```

Comment (Pepsilawn) [reply fulp_modules/features/antagonists/infiltrators/infiltrator.dm:43]: Commited from my phone

**Comment by SgtHunk** [comment]
i don't trust that this won't install 500 bitcoin miners into the server

**Comment by Pepsilawn** [comment]
this will be a new hallmark for the codebase... you shouldnt let the opportunity go

**Pull request merged**